src/v2/Apps/Artist/Components/ArtistCollectionsRail/ArtistCollectionsRail.tsx

```diff
@@ -44,6 +44,8 @@ export class ArtistCollectionsRail extends React.Component<
 
   render() {
     const { collections } = this.props
+    const isMobile = sd?.IS_MOBILE
```

Comment:
This will break during SSR pass, we should use the `getENV("IS_MOBILE") === "true"` helper.